agent/runtime-attach/src/main/java/com/microsoft/applicationinsights/attach/ApplicationInsights.java

```diff
@@ -69,6 +73,15 @@ private static Optional<String> findJsonConfig() {
     }
   }
 
+  private static String findJsonConfigFile() {
+    String fileFromProperty =
+        System.getProperty(APPLICATIONINSIGHTS_RUNTIMEATTACH_CONFIGURATION_FILE);
+    if (fileFromProperty != null) {
```

Comment:
i think we should check if runtime attach is used here?

Reply:
`RUNTIME_ATTACHED_ENABLED_PROPERTY` is always true at this point. This java agent uses this property to detect the runtime attachment and set the appropriate sdk prefix.

Reply:
yup, i didn't see line 44 earlier

Reply:
we might need to check if such a file exists?

Reply:
This PR was created because a user needs a connection string different for each spring profile/environment (see [#2576](https://github.com/microsoft/ApplicationInsights-Java/issues/2576)), with runtime attachment. If the file is not found then, Application Insights will fail to startup with a message saying that a connection string is missing. The user should easily understand that the file containing the connection string was not found?

Reply:
I think a message that the resource was not found could be helpful, we do similar for json configuration file:

https://github.com/microsoft/ApplicationInsights-Java/blob/568a2373dc479181b214a7278aeb1dabe35e5a32/agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java#L455

Reply:
Added since it seems the team preference